pytorch_pfn_extras/training/manager.py

```diff
@@ -340,6 +317,17 @@ def extend(
         if self._start_extensions_called:
             raise RuntimeError(
                 'extend called after the extensions were initialized')
+
+        if isinstance(extension, extension_module.ExtensionEntry):
+            if name is not None:
+                raise ValueError('"name" is duplicated')
+            if trigger is not None:
+                raise ValueError('"trigger" is duplicated')
+            if priority is not None:
+                raise ValueError('"prioirity" is duplicated')
```

Comment:
I think the new values should override the given ExtensionEntry attributes.
(Also take care of `call_before_training`)